core/core.go

```diff
@@ -109,6 +111,11 @@ func NewTextile(ctx context.Context, conf Config) (*Textile, error) {
 		return nil, err
 	}
 
+	fcClient, err := fc.NewClient(conf.AddrFilecoinApi)
```

Comment:
I can't remember, does this have a `Close` method?

Reply:
If so, we should probably close it and `threadsClient` (oops) in the Close method below.

Reply:
Yep, good catch. I'll add that and close the `threadsClient` as well.

Reply:
Ok, I added the `fcClient.Close()` to the `server.Close()` since the `Server` manages the fc client and `Textile` keeps no reference to it. I added `t.threadsClient.Close()` to `Textile.Close()`.